src/pages/Home.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable max-len */
```

Comment:
- [ ] Please remove this eslint disable rule.

Reply:
done